skimage/measure/_regionprops.py

```diff
@@ -358,6 +407,94 @@ def __eq__(self, other):
         return True
 
 
+def _props_to_dict(regions, properties=('label', 'bbox'), separator='-', always_include_label=True):
+    """Convert image regions properties into a dictionary
+
+    Parameters
+    ----------
+    regions : (N,) list
+        List of _RegionProperties objects as returned by :func:`regionprops`
+    properties : tuple, optional
+        properties which the output need to have
+    separator : str, optional
+        Each element of non-scalar properties and is not listed in object_columns will
+        be put into its own column, with the index of that element separated from
+        the name by this separator.
+    always_include_label : bool, optional
```

Comment:
Unclear what this is used for.

Reply:
label. @stefanv my rationale here was that, regardless of what the user passes, `label` is the natural index for the dataframe, and users might not think to include it in the "requested" properties, so we always add it to that list, *unless* you turn that off with this flag. You might want to turn it off if you only care about aggregate statistics rather than label values.

Reply:
Maybe just always include it, until we hear complaints?

Reply:
I thought about that, but found it a bit obnoxious. I might prefer to err on the side of *never* including it, unless it's in the list, and documenting that. Really not sure though!

Reply:
@leGIT-bot let's go ahead and remove the automatic adding altogether (simplifies the internal code *and* the function signature), along with a comment under `properties` that users should remember to add `label` to the desired columns to keep track of object identities.